components/reddit/reddit.app.js

```diff
@@ -3,7 +3,7 @@ const get = require("lodash/get");
 const retry = require("async-retry");
 
 module.exports = {
-  type: "app",
+  type: "app",  
```

Comment:
Can you remove these trailing whitespaces please?

If you're using VSCode, I recommend having this in your `settings.json` file so that your editor removes them automatically:

```json
{
  "files.trimTrailingWhitespace": true
}
```

Reply:
Yes, I went ahead and setup VSCode's settings.json file with the property, and then resaved all this PR's files.

Reply:
The extra whitespaces are still there. Have you committed and pushed the changes to this branch?